electron/electron.js

```diff
@@ -10,14 +9,31 @@ let mainWindow;
 
 function createWindow() {
   mainWindow = new BrowserWindow({
-    width: 300, 
-    height: 130, 
+    webPreferences: {
+      nodeIntegration: true,
+      nativeWindowOpen: true,
+    },
+    width: 300,
+    height: 130,
     show: false,
     titleBarStyle: 'hiddenInset',
     alwaysOnTop: true,
   });
+  mainWindow.webContents.on('new-window', (event, url, frameName, disposition, options, additionalFeatures) => {
```

Comment:
this block here ensures a new window with this configuration can be opened (for use with terminal in the future, or any other thing)